examples/text-classification/run_glue.py

```diff
@@ -151,9 +151,11 @@ def main():
     def build_compute_metrics_fn(task_name: str) -> Callable[[EvalPrediction], Dict]:
         def compute_metrics_fn(p: EvalPrediction):
             if output_mode == "classification":
-                preds = np.argmax(p.predictions, axis=1)
+                preds = p.predictions[0] if type(p.predictions) == tuple else p.predictions
```

Comment:
This line is repeated. You can move it above the if clause.

You also have an 
if
elif

block with no else,

so preds can currently be None on line 159.

Reply:
If there are only two output modes it should be `if/else`